src/operators.js

```diff
@@ -8,6 +8,7 @@ const unaryOperators = {
   typeof: true,
   void: true,
   delete: true,
+  await: true,
```

Comment:
Yeah it is being used. The flow is a little non-obvious though. Basically adding `await` to `unaryOperators` allows [`enforestUnaryExpression`](https://github.com/sweet-js/sweet-core/blob/cdf3c03529057b4f8893fb6ffc066e40224bff2d/src/enforester.js#L2297) to parse `await` like the [unary operator](https://tc39.github.io/ecma262/#sec-unary-operators) it is. Later [`expandUnaryExpression`](https://github.com/sweet-js/sweet-core/blob/cdf3c03529057b4f8893fb6ffc066e40224bff2d/src/term-expander.js#L527-L531) converts `UnaryExpression`s with `await` to a proper AwaitExpression`.

Since sweet only parses modules `await` is not really a contextual keyword; it is always a keyword but only legal inside `async` functions. I not doing that validation but I suppose I probably should.